src/main/java/frc/team4373/robot/commands/wheelspinner/RevolutionSpinCommandGroup.java

```diff
@@ -0,0 +1,18 @@
+package frc.team4373.robot.commands.wheelspinner;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+import frc.team4373.robot.RobotMap;
+
+public class RevolutionSpinCommandGroup extends CommandGroup {
+    /**
+     * Constructs a new command group for deploying the wheel spinner and spinning
+     * the wheel spinner a number of revolutions.
+     * @param revolutions the number of revolutions to spin the wheel.
+     */
+    public RevolutionSpinCommandGroup(int revolutions) {
+        addSequential(new DeploySpinnerCommand());
+        addSequential(new WheelSpinnerRevolutionsCommand(revolutions));
+        addSequential(new RetractSpinnerCommand());
+    }
```

Comment:
We will probably need to drive forward after deploying the spinner before spinning, and drive backward before retracting it.